projects/canopy/src/lib/link-menu/link-menu-item-text/link-menu-item-text.component.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { MockComponents } from 'ng-mocks';
+
+import { LgIconComponent } from '../../icon';
+
+import { LgLinkMenuItemTextComponent } from './link-menu-item-text.component';
+
+describe('LgLinkMenuItemTextComponent', () => {
+  let component: LgLinkMenuItemTextComponent;
+  let fixture: ComponentFixture<LgLinkMenuItemTextComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ LgLinkMenuItemTextComponent, MockComponents(LgIconComponent) ],
+    }).compileComponents();
+  });
+
+  describe('component', () => {
+    beforeEach(() => {
+      fixture = TestBed.createComponent(LgLinkMenuItemTextComponent);
+      component = fixture.componentInstance;
+      fixture.detectChanges();
+    });
+
+    it('should create', () => {
+      expect(component).toBeTruthy();
+    });
+
+    it('should contain the default class', () => {
+      expect(fixture.nativeElement.getAttribute('class')).toContain(
+        'lg-link-menu-item-text',
+      );
+    });
```

Comment:
We should add a test for the `isBold` input too

Reply:
added 2 tests for this 👍